ui/src/importer/importerTaskDetails/ImportedDocuments.js

```diff
@@ -43,6 +81,10 @@ export class ImportedDocuments extends React.Component {
     this.checkForData(taskId);
   }
 
+  componentDidUpdate() {
+    this.filteredRecords();
```

Comment:
Isn't the `this.filteredRecords()` updating the state? If so, then I think you would be in a infinite loop of updates no?

Reply:
Yes, an oversight from my part, I will fix it.